docs/services/sdk/error-handling.mdx

```diff
@@ -19,7 +25,25 @@ import { TerminalError } from "@restatedev/restate-sdk";
 throw new TerminalError("Something went wrong.", { errorCode: 13, cause: "Something caused this." })
 ```
 
-The `errorCode` and `cause` are optional. You can throw a terminal error anywhere in your handler.
+The `errorCode` and `cause` are optional. 
+
+</TabItem>
+<TabItem value="java" label="Java">
+
+You can throw a terminal error via `io.grpc.StatusRuntimeException` or `io.grpc.StatusException`:
```

Comment:
This was changed, now we have terminal exception: https://github.com/restatedev/sdk-java/pull/145